spec/amp-var-substitutions.md

```diff
@@ -622,6 +623,35 @@ Provides the horizontal scroll boundary that triggered a scroll event. This vari
 * **platform variable**: N/A
 * **amp-analytics variable**: `${horizontalScrollBoundary}`
 
+#### HTML Attributes
+
+Provides values of attributes of HTML elements inside of an amp-ad tag which match a given CSS selector. 
+This is only available within the same amp-ad tag; it will not, for instance, allow a publisher to obtain information
```

Comment:
I'm stumbling on the wording "same amp-ad tag".  Same as what? Can you clarify?
I'm wondering if you want to say "this is confined to the contents of the amp-ad tag;"

Reply:
Thanks - I agree that's awkward. What I want to say is:
If an amp-ad tag contains an amp-analytics tag, you can use this within the amp-analytics tag to see the content of the HTML that gets loaded by that particular amp-ad tag. This applies even if the amp-analytics tag uses iframe transport.
But, an amp-analytics tag inside of one amp-ad tag can't use this to see the HTML that gets loaded by a different amp-ad tag. Also, an amp-analytics tag that's not within any amp-ad tag can't use this at all.

Reply:
Ok, I took another shot at it :)